### PR TITLE
3.next Update deprecated config() calls.

### PR DESCRIPTION
**Pull request by markstory**
Update internal uses of deprecated methods with new methods. I've not updated `config()` on the `TableLocator` as `getConfig`/`setConfig()` are not part of that interface.

Refs #10128

### CODE REVIEW

File: src/Error/Debugger.php [hunk 1]
```diff
@@ -192,10 +192,10 @@ public static function getInstance($class = null)
     public static function configInstance($key = null, $value = null, $merge = true)
     {
         if (is_array($key) || func_num_args() >= 2) {
-            return static::getInstance()->config($key, $value, $merge);
+            return static::getInstance()->setConfig($key, $value, $merge);
```
Comment (saeideng) [inline src/Error/Debugger.php]: Maybe  ->setConfig()->getConfig():

**Review by antograssiot — APPROVED** [review]
All those changes looks good to me

File: src/ORM/LazyEagerLoader.php [hunk 2]
```diff
@@ -96,9 +96,9 @@ protected function _getQuery($objects, $contain, $source)
             ->contain($contain);
 
         foreach ($query->getEagerLoader()->attachableAssociations($source) as $loadable) {
-            $config = $loadable->config();
+            $config = $loadable->getConfig();
             $config['includeFields'] = true;
```
Comment (saeideng) [inline src/ORM/LazyEagerLoader.php]: Maybe setConfig('includeFields',true)

**Comment by chinpei215** [comment]
Looks good to me.

File: src/ORM/LazyEagerLoader.php (hunk 2 above)
Comment (markstory) [inline src/ORM/LazyEagerLoader.php]: The implementation on `EagerLoadable` doesn't support that signature.

File: src/Error/Debugger.php (hunk 1 above)
Comment (saeideng) [inline src/Error/Debugger.php]: SetConfig returned @this :)

**Pull request merged**